### PR TITLE
task antisnake complete

### PR DESCRIPTION
**Pull request by andlap24**
[DEMO LINK](https://andlap24.github.io/layout_antisnake/)
[TEST REPORT LINK](https://andlap24.github.io/layout_antisnake/report/html_report/)

### CODE REVIEW

**Review by Feuerbach98 — CHANGES_REQUESTED** [review]
The width of the blocks should behave the same as in the snake task. Fix it and write me about the second review.

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body>
+    <div class="antisnake">
+      <div class="antisnake__block">1</div>
+      <div class="antisnake__block--2">2</div>
+      <div class="antisnake__block--3">3</div>
+      <div class="antisnake__block--4">4</div>
+      <div class="antisnake__block--5">5</div>
+      <div class="antisnake__block--6">6</div>
```
Comment (Feuerbach98) [inline src/index.html:17]: You cannot use modifier without element class

File: src/styles/main.scss [hunk 2]
```diff
@@ -1,3 +1,52 @@
+@import "reset";
+
 body {
-  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.antisnake {
```
Comment (Feuerbach98) [inline src/styles/main.scss:7]: You should create file for every BEM block

File: src/styles/main.scss [hunk 3]
```diff
@@ -1,3 +1,52 @@
...
+  justify-content: center;
+  text-align: center;
+
+  color: white;
+  font-size: 100px;
+  line-height: 300px;
+
+  &__block {
+    height: 300px;
+    width: 300px;
```
Comment (Feuerbach98) [inline src/styles/main.scss]: it do nothing

File: src/styles/main.scss [hunk 4]
```diff
@@ -1,3 +1,52 @@
...
-  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.antisnake {
+  display: grid;
+  grid-template-columns: repeat(3, 300px);
+  gap: 0;
+
+  justify-content: center;
```
Comment (Feuerbach98) [inline src/styles/main.scss:12]: 12 line is not needed

File: src/styles/main.scss [hunk 5]
```diff
@@ -1,3 +1,52 @@
...
+
 body {
-  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.antisnake {
+  display: grid;
+  grid-template-columns: repeat(3, 300px);
+  gap: 0;
```
Comment (Feuerbach98) [inline src/styles/main.scss:10]: it is default value of grid-gap

File: src/styles/main.scss [hunk 6]
```diff
@@ -1,3 +1,52 @@
...
+    background-color: rgb(102, 0, 0);
+  }
+
+  &__block--5 {
+    background-color: rgb(51, 0, 0);
+  }
+
+  &__block--6 {
+    background-color: rgb(0, 0, 0);
+  }
```
Comment (Feuerbach98) [inline src/styles/main.scss:43]: Try to automate the generation of classes for modifiers (optional)

**Review by Feuerbach98 — CHANGES_REQUESTED** [review]

File: src/styles/main.scss (hunk 6 above)
Comment (andlap24) [reply src/styles/main.scss:43]: right decision, if we have some more blocks, with my solution we should add more code lines for new blocks. I will use @for statement. Thanks

File: src/styles/main.scss (hunk 5 above)
Comment (andlap24) [reply src/styles/main.scss:10]: yep, 10 line no needed

**Review by Feuerbach98 — APPROVED** [review]
Good job!

File: src/styles/grid-container.scss [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+  color: white;
+  line-height: 300px;
+
+  &__block {
+    min-width: $blockSize;
+    text-align: center;
+
+    @for $i from 0 through 5 {
+      &:nth-child(#{$i + 1}) {
+        background-color: rgb($redShade - ($shadeStep * $i), 0, 0);
```
Comment (solaryasha) [inline src/styles/grid-container.scss:14]: Крута формула !

**Review by solaryasha — APPROVED** [review]

**Pull request closed without merging**